symphony/bdk/core/retry/strategy.py

```diff
@@ -3,41 +3,70 @@
 
 from symphony.bdk.core.auth.auth_session import AuthSession
 from symphony.bdk.core.auth.exception import AuthUnauthorizedError
+from symphony.bdk.core.service.datafeed.abstract_datafeed_loop import AbstractDatafeedLoop
 from symphony.bdk.gen import ApiException
 
 
-def is_unauthorized(exception: ApiException) -> bool:
-    return exception.status == 401
+def is_unauthorized(exception: Exception) -> bool:
+    if isinstance(exception, ApiException):
```

Comment:
This can even be transformed into a one-liner : `return isinstance(exception, ApiException) and exception.status == 401`.